loopin.go

```diff
@@ -54,6 +54,10 @@ type loopInSwap struct {
 
 	htlc *swap.Htlc
 
+	htlcP2WSH *swap.Htlc
+
+	htlcNP2WSH *swap.Htlc
```

Comment:
Further decomposition of the commit structure would be to first only add `htlcNP2WSH`, but maybe you think that is a step too far.

Reply:
I think it would be an improvement for review.